docs/tutorials/avro.ipynb

```diff
@@ -0,0 +1,379 @@
+{
```

Comment:
<p>I think reading and printing the content of the schema file is a better approach. Helps in avoiding content mismatch.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>